package.json

```diff
@@ -128,9 +128,9 @@
     "split.js": "^1.5.11",
     "ts-jest": "^24.2.0",
     "ts-node": "^7.0.1",
-    "tslint": "^5.20.1",
+    "tslint": "^6.1.2",
     "typedoc": "^0.15.0",
-    "typescript": "^3.7.4",
+    "typescript": "^3.8.0",
```

Comment:
The reason why this change was made is because this PR makes use of `export * as ns`, which was a feature added in TS 3.8:
https://github.com/zowe/imperative/blob/e2b3716c648f6a2e9a6257ffc62c0ba9b54b8a9d/packages/rest/index.ts#L22

This is also why TSLint was updated from 5.x to 6.x. Version 5.x threw an error when it parsed `export * as ns` because it didn't recognize it as valid syntax.

Reply:
Added typescript and tslint updates to changelog.